SU-HSDO/stanford_mrc
# MRCD8-243 rm sub nav styles

# READY FOR REVIEW

# Summary
- Added sub-navigation styles for hover menu molecule
- Removed original couple of rules from remove_me file

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the subnav looks good now that there is no JS associated with it and down to three levels are exposed
3. Compare to subnav mock at https://invis.io/DCEGPUZS4#/263924862_News_Node_Display

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-243

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
